test/org/zaproxy/zap/extension/ascanrules/CommandInjectionPluginUnitTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2016 The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.ascanrules;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.apache.commons.configuration.Configuration;
+import org.junit.Test;
+import org.zaproxy.zap.utils.ZapXmlConfiguration;
+
+/**
+ * Unit test for {@link CommandInjectionPlugin}.
+ */
+public class CommandInjectionPluginUnitTest extends ActiveScannerTest<CommandInjectionPlugin> {
+
+    @Override
+    protected CommandInjectionPlugin createScanner() {
+        CommandInjectionPlugin scanner = new CommandInjectionPlugin();
+        scanner.setConfig(new ZapXmlConfiguration());
+        return scanner;
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldFailToInitWithoutConfig() throws Exception {
+        // Given
+        CommandInjectionPlugin scanner = new CommandInjectionPlugin();
+        // When
+        scanner.init(getHttpMessage(""), parent);
+        // Then = NullPointerException
+    }
+
+    @Test
+    public void shouldNotFailToInitWithConfig() throws Exception {
```

Comment:
This test is the opposite of the previous one, that's why I used the not. I'm fine changing to whatever is more readable though.